lesson_2.ipynb

```diff
@@ -0,0 +1,2129 @@
+{
```

Comment:
<p>Great point at how connecting the dots is not a good approach. Also if our method was to connect the dots how would we estimate uncertainty?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='108'/>